filters/ThirdParty/filter_204_PeterLowesList/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 204,
   "name": "Peter Lowe's list",
```

Comment:
title as well

Reply:
why? It is called differently in all places.

Reply:
`Malware Domains` also has no own title.

Reply:
@Alex-302 As to why: The title is because Peter always names his own as a 'blocklist' as per the source provided. Malware Domains also called itself 'Malware Domain Blocklist' as per their title in the source provided. (cc: @ameshkov) Thought it would be good to match the names as per the author's method.